src/app/containers/ArticleMain/index.jsx

```diff
@@ -21,11 +22,14 @@ const ArticleMain = ({ articleData }) => {
   const { content, metadata, promo } = articleData;
   const { blocks } = content.model;
 
+  const { audiovideo } = useContext(DialContext);
```

Comment:
Very pedantic here, but I would name this something like `audioVideoEnabled` feels a little more descriptive as to what is actually returned

Reply:
Good idea. The dial is named `audiovideo` for consistency with the existing one which is called `mpulse`. I've renamed the temp variable to make this code easier to follow.